### PR TITLE
PTX-2725 Taking images from list incase StorageCluster doesn't have images defined

### PR DESCRIPTION
**Pull request by nikolaypopov**
Taking images from list incase StorageCluster doesn't have images defined

### CODE REVIEW

File: pkg/util/test/util.go [hunk 1]
```diff
@@ -521,7 +522,18 @@ func expectedPods(cluster *corev1.StorageCluster) (int, error) {
 	return podCount, nil
 }
 
-func validateComponents(cluster *corev1.StorageCluster, timeout, interval time.Duration) error {
+func getImageNameFromMap(pxImageList map[string]string, component string) string {
```
Comment (thiguetta) [inline pkg/util/test/util.go:525]: why do we need this? can we simple use `image := pxImageList[component]`?

Comment (thiguetta) [reply pkg/util/test/util.go:525]: if you need to validade if the key is present you can use
```
if value, ok := pxImageList[component]; ok {
         image  =  value
}
```

Comment (nikolaypopov) [reply pkg/util/test/util.go:525]: Thats a good idea @thiguetta let me try this out!

**Review by thiguetta — APPROVED** [review]
LGTM

**Pull request merged**